docs/guides/admin/docs/workflowoperationhandlers/send-email-woh.md

```diff
@@ -78,6 +78,29 @@ Use `${workflow.FIELD}`
 
 Note that some of these values may not exist depending on the workflow.
 
+## Failed Operation Information
+
+|Field                    |How To Get It                                     |
+|-------------------------|--------------------------------------------------|
+|id                       |${failedOperation.getId}                          |
```

Comment:
Hmm I'm not sure you need the "get" in these. Have you tried with failedOperation.id, failedOperation.dateStarted, etc?

Reply:
Thanks Rute for checking it. I just tried every combination and I left the only ones that worked with `freemaker`